docs/src/docs/modules/layout-check.adoc

```diff
@@ -15,10 +15,6 @@ At this point an automatic comparison of reference screenshots can help. This te
 == Description
 
 A layout test with the Testerra utilities is actually a comparison between a reference and the actual state. This is done via the screenshot functionality of Selenium, in which both screenshots (reference and actual) are compared pixel by pixel from top to bottom. In this comparison, a third image is created in which differences (ReferencePixel != ActualPixel) are marked in red.
-The LayoutCheck can perform a check in two ways:
-
-- *PIXEL*: Pixel-based image comparison of the entire reference image without exception. You get a quantified value for pixel displacement in percent (0-100%)
-- *ANNOTATED*: The image comparison is based on annotations in the reference image. The annotations are rectangles in a certain color, which include the areas to be checked. You get a quantified value for displaced annotations in percent.
 
 === Prerequisites
 The following prerequisites must be met
```

Comment:
This part still contains some 2 outdated parts:
* l. 24: There is used the attribute `{default_resolution}` which is located in `common-attributes.adoc`. In my optinion this file is not needed any more. Please remove it with its include in `index.adoc` and insert the default resolution `1290x1080`
* l.25: `Build server: The library...` describes requirements for ANNOTATED mode
*